chia/full_node/full_node.py

```diff
@@ -1172,13 +1179,11 @@ async def validate_block_batches(
                             assert fork_hash is not None
                             fork_info = ForkInfo(fork_point_height - 1, fork_point_height - 1, fork_hash)
 
-                success, state_change_summary, ssi, diff, prev_ses_block, err = await self.add_block_batch(
+                success, state_change_summary, err = await self.add_block_batch(
                     blocks,
                     peer.get_peer_logging(),
                     fork_info,
-                    ssi,
-                    diff,
-                    prev_ses_block,
+                    cs,
```

Comment:
i think we need a comment or something to make sure its understood that cs is updated inside as we iterate the batches